telegram/chatmemberupdated.py

```diff
@@ -113,3 +113,47 @@ def to_dict(self) -> JSONDict:
         data['date'] = to_timestamp(self.date)
 
         return data
+
+    def difference(
+        self,
+    ) -> Dict[
+        str,
+        Tuple[
+            Union[str, bool, datetime.datetime, User], Union[str, bool, datetime.datetime, User]
+        ],
+    ]:
+        """Computes the difference between :attr:`old_chat_member` and :attr:`new_chat_member`.
+
+        Example:
+            .. code:: python
+
+                >>> chat_member_updated.difference()
+                {'custom_title': ('old title', 'new title')}
+
+        Note:
+            To determine, if the :attr:`telegram.ChatMember.user` attribute has changed, *every*
+            attribute of the user will be checked.
+
+        Returns:
+            Dict[:obj:`str`, Tuple[:obj:`obj`, :obj:`obj`]]: A dictionary mapping attribute names
+            to tuples of the form ``(old_value, new_value)``
```

Comment:
```suggestion
            to tuples of the form ``(old_value, new_value)``
            
            .. versionadded:: 13.5
```

Missed this (I guessed from GitHub where to add it, I could be wrong, double check pls)